pkg/volume/csi/csi_plugin_test.go

```diff
@@ -41,8 +41,26 @@ import (
 	volumetest "k8s.io/kubernetes/pkg/volume/testing"
 )
 
-// create a plugin mgr to load plugins and setup a fake client
+const (
+	volumeHostType int = iota
```

Comment:
I'm unsure if this is a great approach. An alternative is passing a FakeVolumeHost constructor as an argument (like `NewFakeVolumeHostWithCSINodeName`, `NewFakeKubeletVolumeHostWithCSINodeName`, etc.)